src/main/java/Duke.java

```diff
@@ -1,10 +1,197 @@
+        String inputWithoutType = String.join(" ", inputArrayWithoutType);
+        if (inputWithoutType.length() < 3){
+            throw new StringIndexOutOfBoundsException();
+        }
+        String description = inputWithoutType.split(" /at ")[0];
+        String time = inputWithoutType.split(" /at ")[1];
+        Event newEvent = new Event(description, time);
+        taskLists.add(newEvent);
+        Task.numOfTasks ++;
+        printEchoInput(newEvent);
+    }
+    // add exception
+
+    private static void addDeadline(ArrayList<Task> taskLists, String[] userInputSplit) {
+        String[] inputArrayWithoutType = Arrays.copyOfRange(userInputSplit,1, userInputSplit.length);
+        String inputWithoutType = String.join(" ", inputArrayWithoutType);
+        String description = inputWithoutType.split(" /by ")[0];
+        String deadline = inputWithoutType.split(" /by ")[1];
+        Deadline newDeadline = new Deadline(description, deadline);
+        taskLists.add(newDeadline);
+        Task.numOfTasks ++;
+        printEchoInput(newDeadline);
+    }
+    // add exception
+
+    private static void printDontKnowMessage(){
+        String dontKnowMessage = "    ____________________________________________________________\n"
+                        + "     ☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n"
+                        + "    ____________________________________________________________";
+        System.out.println(dontKnowMessage);
+    }
+
+    public static String getUserInput(){
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        return line;
+    }
+
+    public static void printEchoInput(Task task){
+        System.out.println("    ____________________________________________________________");
+        System.out.println("     Got it. I've added this task:");
+        System.out.println(String.format("       %s", task.toString()));
+        System.out.println(String.format("     Now you have %d tasks in the list.", Task.numOfTasks));
+        System.out.println("    ____________________________________________________________");
+    }
+
+    public static void printWelcomeMessage() {
+        String welcomeMessage =
+                "    ____________________________________________________________\n"
+                + "     Hello! I'm Duke\n"
+                + "     What can I do for you?\n"
+                + "    ____________________________________________________________";
+        System.out.println(welcomeMessage);
+    }
+
+    public static void printByeMessage() {
+        String byeMessage =
+                "    ____________________________________________________________\n"
+                + "     Bye. Hope to see you again soon!\n"
+                + "    ____________________________________________________________";
+        System.out.println(byeMessage);
+    }
+
+    // replace count
```

Comment:
Write comments to the reader. Can remove unnecessary comments if they are not to the reader.

Coding Standard Guideline: Comment minimally, but sufficiently